App.js

```diff
@@ -1,6 +1,7 @@
 import { StatusBar as ExpoStatusBar } from "expo-status-bar";
 import React from "react";
 import { ThemeProvider } from "styled-components/native";
+import * as firebase from "firebase";
```

Comment:
If you are on firebase 9 please checkout master for a more up to date reference:
https://github.com/mobinni/MealsToGo/blob/master/App.js